src/views/bibliography_item.tsx

```diff
@@ -0,0 +1,172 @@
+/*!
+ * © 2022 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BibliographyItem, Model } from '@manuscripts/json-schema'
+import {
+  CitationProvider,
+  createBibliographyElementContents,
+  loadCitationStyle,
+} from '@manuscripts/library'
+import {
+  Build,
+  DEFAULT_BUNDLE,
+  ManuscriptNodeView,
+} from '@manuscripts/transform'
+
+import { sanitize } from '../lib/dompurify'
+import { BaseNodeProps, BaseNodeView } from './base_node_view'
+import { createNodeView } from './creators'
+import React from 'react'
+
+const createBibliography = async (items: BibliographyItem[]) => {
+  const styleOpts = { bundleID: DEFAULT_BUNDLE }
+  const citationStyle = await loadCitationStyle(styleOpts)
+  const [bibmeta, bibliographyItems] =
+    CitationProvider.makeBibliographyFromCitations(items, citationStyle)
+
+  if (bibmeta.bibliography_errors.length) {
+    console.error(bibmeta.bibliography_errors)
+  }
+  const contents = createBibliographyElementContents(bibliographyItems)
+  return contents
+}
+
+interface BibliographyItemViewProps extends BaseNodeProps {
+  filterLibraryItems: (query: string) => Promise<BibliographyItem[]>
+  saveModel: <T extends Model>(model: T | Build<T> | Partial<T>) => Promise<T>
+  deleteModel: (id: string) => Promise<string>
+  setLibraryItem: (item: BibliographyItem) => void
+  removeLibraryItem: (id: string) => void
+  modelMap: Map<string, Model>
+  components: Record<string, React.ComponentType<any>> // eslint-disable-line @typescript-eslint/no-explicit-any
+}
+
+export class BibliographyItemView<PropsType extends BibliographyItemViewProps>
+  extends BaseNodeView<PropsType>
+  implements ManuscriptNodeView
+{
+  protected popperContainer?: HTMLDivElement
+
+  public showPopper = (referenceID: string) => {
+    const {
+      filterLibraryItems,
+      saveModel,
+      deleteModel,
+      setLibraryItem,
+      removeLibraryItem,
+      renderReactComponent,
+      modelMap,
+      components: { ReferencesEditor },
+    } = this.props
+
+    if (!this.popperContainer) {
+      this.popperContainer = document.createElement('div')
+      this.popperContainer.className = 'references'
+    }
+
+    renderReactComponent(
+      <ReferencesEditor
```

Comment:
can we use `CitationEditor` here as it's like the view we show for citation edit, or I'm missing something?

Reply:
The `CitationEditor` and the `ReferencesEditor` created on https://github.com/Atypon-OpenSource/manuscripts-article-editor/pull/146 are two different things.

By the `ReferencesEditor` we just need to show the references model, not the edit citations model.